### PR TITLE
Narrow node engine version

### PR DESCRIPTION
**Pull request by wemeetagain**
Temporary fix until #1370 is resolved

### CODE REVIEW

**Review by GregTheGreek — DISMISSED** [review]

**Comment by mpetrunic** [comment]
I'm running it with 12.14.1 for last 3-4 months so I don't think 12.13 is issue. in channel someone built it with 12.13 which you've excluded

**Comment by wemeetagain** [comment]
@mpetrunic Oh, right. should we use <12.18.0?

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "root",
   "private": true,
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=12.0.0 <12.13.0"
```
Comment (wemeetagain) [inline package.json]:
```suggestion
    "node": ">=12.0.0 <12.18.0"
```

**Review by dapplion — DISMISSED** [review]

**Comment by mpetrunic** [comment]
> @mpetrunic Oh, right. should we use <12.18.0?

I would go <=12.14.1 since it's highest confirmed working version. You could try 12.17 if you wanna pinpoint version with issue :smile:

File: package.json (hunk 1 above)
Comment (wemeetagain) [inline package.json]:
```suggestion
    "node": ">=12.0.0 <=12.14.1"
```

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**